packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.scss

```diff
@@ -189,3 +189,61 @@ $gux-icon-radio-checked: "data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='h
     @include mixins.gux-sr-only;
   }
 }
+
+:host(.gux-descriptive) {
```

Comment:
I think it is a bad idea to have these classes applied to the host element.
Is there a reason we need to use the `Host` element?